ONSdigital/srm-rh-ui
# 1081 rh UI implement security middleware and security policies

# Motivation and Context
Security headers needed to be set in new UI

# What has changed
Added security headers

# How to test?
Build and deploy to your env, and run both old & new UI against https://securityheaders.com
Result should be the same other than `X-Content-Security-Policy` missing

# Screenshots (if appropriate):
Old:
<img width="2048" alt="ssdc-int-rh-ui result" src="https://github.com/ONSdigital/srm-rh-ui/assets/23309814/4099c09d-c2ed-4061-bc0f-aaa42654f19e">
New:
<img width="2048" alt="srm-rh-ui result" src="https://github.com/ONSdigital/srm-rh-ui/assets/23309814/ee0fd219-c34a-4ab7-93af-e7e9b06a505d">
